docs/_static/theme_overrides.css

```diff
@@ -34,6 +34,14 @@ abbr {
     text-decoration : none !important;
 }
 
+.sd-card-title {
+    font-size: 0.75em;
+}
+
+.sd-card-text {
+    font-size: 0.75em;
```

Comment:
```suggestion
    font-size: 0.875em;
}

.sd-card-text {
    font-size: 0.875em;
```
you're going to hate me for this, but IMO 14px looks even nice
![image](https://github.com/dwavesystems/dwave-ocean-sdk/assets/8395238/5c0f7b50-eb50-493d-89c4-da1f81a960a1)
![image](https://github.com/dwavesystems/dwave-ocean-sdk/assets/8395238/3d3171c7-57ef-46a9-83bd-01491aaca689)